apis/v1beta1/collector_webhook_test.go

```diff
@@ -35,34 +41,48 @@ import (
 	"k8s.io/client-go/kubernetes/fake"
 	"k8s.io/client-go/kubernetes/scheme"
 	kubeTesting "k8s.io/client-go/testing"
-
-	"github.com/open-telemetry/opentelemetry-operator/internal/config"
-	"github.com/open-telemetry/opentelemetry-operator/internal/rbac"
 )
 
 var (
 	testScheme = scheme.Scheme
 )
 
+var cfgYaml = `receivers:
```

Comment:
This is used just in the `TestOTELColValidatingWebhook` method, right? Can we move it to there?